website/templates/feed.html

```diff
@@ -30,6 +30,8 @@ <h4>{{ trail.date.strftime("%A, %B %d, %Y") }}</h4>
                 <p>Notes: {{ trail.notes }}</p>
               </p>
                 <a href="http://maps.google.com/?q={{ trail.location }}" class="btn" id="location-button">Location</a>
+                <!-- TODO add a button here that either opens a modal or takes them to a photo album 
```

Comment:
Ah; so this is DOM manipulation.

If you're using something like jQuery, you could do the following:

```html
<button id="modal-open">Look at your photos</button>
```
And in the jQuery:
```javascript
$('#modal-open').on('click', () => {
  // do stuff to show the modal
})

Reply:
Thank you so much for this response. I think my bigger question that I need help with is figuring out the logic to have that button open a photo album of all the photos associated with only that particular trail.